doc/whats-new.rst

```diff
@@ -10,18 +10,33 @@ v0.6.0 (unreleased)
 
 New Features
 ~~~~~~~~~~~~
-- :py:meth:`~xgcm.grid.Grid.interp_like` enables users to interpolate arrays onto the grid positions of another array (:issue:`234` , :issue:`343`, :pull:`350`). By `Dianne Deauna <https://github.com/jdldeauna>`_ under the `SIParCS internship <https://www2.cisl.ucar.edu/siparcs-2021-projects#8>`_.
+- :py:meth:`~xgcm.grid.Grid.get_metric` refactored, and now incorporates :py:meth:`~xgcm.grid.Grid.interp_like` to allow for automatic interpolation of missing metrics from available values on surrounding positions (:pull:`345`, :pull:`354`).
+  By `Dianne Deauna <https://github.com/jdldeauna>`_.*
 
-- :py:meth:`~xgcm.grid.Grid.set_metrics` now enables adding metrics to a grid object (:pull:`336`, :issue:`199`).
-  By `Dianne Deauna <https://github.com/jdldeauna>`_ under the `SIParCS internship <https://www2.cisl.ucar.edu/siparcs-2021-projects#8>`_.
+- :py:meth:`~xgcm.grid.Grid.set_metrics` enables overwriting of previously assigned metrics to a grid object, and allows for multiple metrics on the same axes (must be different dimensions) (:pull:`351`, :issue:`199`).
+  By `Dianne Deauna <https://github.com/jdldeauna>`_.*
+
+- :py:meth:`~xgcm.grid.Grid.interp_like` enables users to interpolate arrays onto the grid positions of another array, and can specify periodic and boundary conditions (:issue:`234` , :issue:`343`, :pull:`350`).
+  By `Dianne Deauna <https://github.com/jdldeauna>`_.*
 
 - Better input checking when creating a grid object avoids creating grid positions on dataset coordinates which are not 1D (:issue:`208`, :pull:`358`).
   By `Julius Busecke <https://github.com/jbusecke>`_.
 
+*under the `SIParCS internship <https://www2.cisl.ucar.edu/siparcs-2021-projects#8>`
+
 Breaking Changes
 ~~~~~~~~~~~~~~~~
 - Drop support for Python 3.6 (:issue:`360`, :pull:`361`). By `Julius Busecke <https://github.com/jbusecke>`_.
 
+Documentation
+~~~~~~~~~~~~~
+- Added section about boundary conditions (:pull:`325`, :issue:`273`).
```

Comment:
I just think this [looks great](https://xgcm--325.org.readthedocs.build/en/325/grids.html) and is super informative, hope it can be merged soon.

Reply:
Wait, if this is not merged, how is it in the Docs? We should only add things that are merged in the master.

Reply:
Oh sorry, I just thought I would advocate for it to be merged by adding it to the whats-new file as well. I realize now that can be confusing, won't do it again.